docs/use-cases/tokenizations.page.tsx

```diff
@@ -0,0 +1,310 @@
+import * as React from "react";
+import { useTranslate } from "@portal/hooks";
+
+export const frontmatter = {
+  seo: {
+    title: "XRPL Overview",
+    description:
+      "An introduction to the key benefits and features of the XRP Ledger, a public blockchain driven by the XRPL developer community.",
+  },
+};
+
+const useCases = [
+  {
+    name: "Stablecoin Issuer",
+    url: "",
```

Comment:
Looks like actual link paths are missing from these entries? The elements they map to look like they're meant to be links but aren't.

Stablecoin Issuer will be this use case:

```suggestion
    url: "/docs/use-cases/tokenization/stablecoin-issuer/",
```